### PR TITLE
Shared monitoring plugins

### PR DESCRIPTION
**Pull request by peterhoeg**
###### Motivation for this change

What was previously called "Nagios Plugins" has now renamed itself to the "Monitoring Plugins" project.

As a consequence, we move it around here as well and introduce a number of other
shared check plugins for use by sensu, nagios, ichinga and others.

- check-ups-health and check-nwc-health have been tested but we are not using them yet
- the renamed monitoring-plugins as well as check-esx-hardware are in use in production on our side

@thoughtpolice @relrod

###### Things done

Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers.

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -29464,6 +29464,25 @@ EOF
     };
   };
 
+  pywbem = buildPythonPackage rec {
```
Comment (Mic92) [inline pkgs/top-level/python-packages.nix]: Please move this expression to `python-modules` and only reference it here.

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -29464,6 +29464,25 @@ EOF
...
+    name  = "pywbem-${version}";
+    version = "0.10.0";
+    src = pkgs.fetchFromGitHub {
+      owner = "pywbem";
+      repo = "pywbem";
+      rev = "v${version}";
+      sha256 = "0jcwklip03xcni0dvsk9va8ilqz21g4fxwqd5kzvv91slaadfcym";
+    };
+    propagatedBuildInputs = with self; [ m2crypto ply pyyaml six ];
+    doCheck = false;
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: include a reason why you disable tests

File: pkgs/top-level/python-packages.nix (hunk 1 above)
Comment (peterhoeg) [inline pkgs/top-level/python-packages.nix]: I did not know that had been split up, thanks!

File: pkgs/top-level/python-packages.nix (hunk 2 above)
Comment (peterhoeg) [inline pkgs/top-level/python-packages.nix]: Actually, now that I have your attention, I was hoping you could clarify something... ;-) If we run the tests, they don't actually run and I am told that 0 tests ran. I have tried adding tox and pytest to ```checkInputs``` but that doesn't change anything. Is there anything obvious I am missing to actually run the tests?

Comment (Mic92) [inline pkgs/top-level/python-packages.nix]: 1. Are the tests distributed on pypi? sometimes they are only in the github release
2. Did you override `checkPhase` with a pytest call?

File: pkgs/top-level/python-packages.nix (hunk 1 above)
Comment (peterhoeg) [inline pkgs/top-level/python-packages.nix]: Done

File: pkgs/top-level/python-packages.nix (hunk 2 above)
Comment (peterhoeg) [inline pkgs/top-level/python-packages.nix]: For some reason running ```make test``` works but setting ```checkTarget = "test";``` doesn't. In any case, test are now in place.

Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: That's likely because `buildPythonPackage` passes its own `checkPhase` to `stdenv.mkDerivation`.

Comment (peterhoeg) [inline pkgs/top-level/python-packages.nix]: Tests are sorted.

**Pull request merged**